code/style.css

```diff
@@ -1,4 +1,309 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*{
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
 }
+
+
+body{
+  width: 100%;
+  min-height: 100%;
+  background-color: #dadada;
+  font-family: Raleway;
+  display: flex;
+  flex-direction: column;
+}
+
+img {
+  width: 90%;
+  height: 90%; 
+}
+
+header{
+display: flex;
+align-items: center;
+width: 100%;
+} 
+
+nav{
+width: 100vw;  
+height: 100px;
+}
+
+.logo{
+  font-family: 'Long Cang';
+  font-size: 60px;
+  color: black;
+  padding: 20px 0px 20px 60px;
+}
+
+.menu-button {
+  display: flex;
+  height: 30px;
+  width: 30px;
+  font-size: 20px;
+  justify-content: flex-end;
+  margin: 10px;
+}
+
+/*removes button styling*/
+button, input[type="submit"], input[type="reset"] {
+	background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
+}
+
+.dropdown {
+  display: flex;
+  justify-content: flex-end;
+  align-content: flex-start; 
+  color: black;
+  padding: 14px;
+  line-height: 40px;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: rgb(197, 197, 197);
+  min-width: 10px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+  padding: 20px 10px;
+  border-radius: 10px;
+}
+
+nav ul li a {
+  color: black;
+  font-size: 20px;
+  padding: 14px 13px;
+  text-transform: uppercase;
+  border-radius: 13px;
+  font-family: Quicksand;
+  }
+
+  a:hover{
+    color: coral;
+  }
+
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: rgb(206, 206, 206);}
+
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {display:list-item;}
+
+/* Change the background color of the dropdown button when the dropdown content is shown */
+.dropdown:hover .dropbtn {background-color: rgb(255, 213, 198);}
+
+.container-1{
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  padding: 10px;
+  box-sizing: border-box;
+}
+
+
+h1 {
+  font-size: 15px;
+  margin: 0px 0px 10px 0px;
+  font-family: Arial, Helvetica, sans-serif
+}
+
+ .container-1 h2 {
+  padding: 30px 0px;
+  font-family: Quicksand;
+  font-size: 20px;
+} 
+
+
+.container-2 h2 {
+  padding: 20px 0px;
+  font-family: Quicksand;
+  font-size: 20px;
+} 
+
+
+.container-1 container-2 p{
+  font-size: 15px;
+  line-height: 32px;
+  font-family: Raleway;
+}
+
+.big-news img{
+  align-items: center;
+  width: 100%;
+  height: auto;
+  margin-inline-start: 20px;
+}
+
+.big-news h1{
+  background-color: black;
+  position: absolute;
+  color: white;
+  bottom: 75%;
+  width: 50%;
+  padding: 20px;
+}
+
+.big-news-text{
+  width: 100%;
+  margin-inline-start: 10px;
+  padding: 40px;
+}
+
+.container-2{
+  display: flex;
+  flex-direction: column;
+}
+
+.container-2 img{
+  width: 100%;
+  height: auto; 
+  margin-inline-start: 20px;
```

Comment:
The only thing I could find on this page that wasn't perfect was the margin for this image on mobile. With the inspector I changed this to 8px and then it seem to be centered on mobile. But 20px is perfect for the other devices.